cypress/e2e/access-keys.cy.js

```diff
@@ -7,6 +7,8 @@ describe("Exercise access keys", () => {
       defaultCommandTimeout: 30000,
     },
     () => {
+      cy.log("***************");
+      cy.log(`HELLLOOO auth url ${Cypress.env("CYPRESS_AUTH_URL")}`);
       cy.log(`CYPRESSENV: ${Cypress.env("AUTH_USERNAME")}`);
```

Comment:
You can delete the cy.logs. You can even delete the “CYPRESSENV” one that I added. Should really get rid of all those. I’d forgotten about them.

Reply:
I can also remove the AUTH_USERNAME log then